lib/importer/sheets.js

```diff
@@ -32,13 +32,12 @@ exports.GetRows = (session, start=0, count=10) => {
     const sheetDimensions = backend.SessionGetInputDimensions(session.backendSid)
           .sheetDimensions.get(session.sheet);
 
-    // We have to calculate the range containing the data based on what we believe
-    // the header range to be.
-    const hRange = session.headerRange;
+    // Try to provide rows from start to start+count-1, but limit ourselves to
+    // the rows in the sheet.
```

Comment:
The previous version used session.headerRange which might be the default, but also could be a selection in the middle of the sheet.  Do we handle that case another way now without using the headerRange range in the session?

Reply:
This is used for the screen where the user selects the header rows, so it needs to display the whole sheet. When you hit it normally, the headerRange is set to a default that means you get to see everything, but the bug this tries to fix is that if you select a header range then navigate back to the selection page and reload it, it then shows less than the entire sheet because there's now a non-default headerRange set. So you can't move the header range back UP if you move it too far down by accident.

Reply:
Awesome, thanks for clarifying